src/Sulu/Bundle/AdminBundle/Tests/Functional/Controller/fixtures/default.json

```diff
@@ -1,12 +1,9 @@
 {
     "name": "default",
-    "title": "Animals",
     "form": {
```

Comment:
we can not set the position of a property added via subscriber this are just moved down